src/McBopomofo.cpp

```diff
@@ -215,6 +254,20 @@ void McBopomofoEngine::activate(const fcitx::InputMethodEntry&,
 
 void McBopomofoEngine::reset(const fcitx::InputMethodEntry&,
                              fcitx::InputContextEvent& event) {
+  auto isFocusOutEvent = dynamic_cast<fcitx::FocusOutEvent*>(&event) != nullptr;
+
+  // Note: when inputting in Chrome based web browsers, the `reset` method would
```

Comment:
Interesting, I never ran into this problem before... could it be that is something in Ubuntu 21.10's Wayland and its interactions with Chrome or fcitx5?

The change here LGTM.